packages/nimbus/src/libs/redux/slices/layoutSlice.ts

```
@@ -27,92 +27,26 @@ interface InteractionBasedLayout {
-            if (visibleTabs.length > 0) {
-                Object.values(state.interactionBasedLayout.activeRequestOptions).forEach(
-                    (option) => (option.selected = false)
-                );
-                state.interactionBasedLayout.activeRequestOptions[visibleTabs[0].id].selected = true;
-            }
-        },
     },
 });
 
-export const { toggleToMockApiLayout, setActiveTab, toggleContainer, changeWorkspace } = layoutSlice.actions;
+export const { } = layoutSlice.actions;
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)